src/src/rx_sx1280_test.cpp

```diff
@@ -0,0 +1,48 @@
+#include <Arduino.h>
+#include "targets.h"
+#include "SX1280Driver.h"
+#include "ESP8266WiFi.h"
+#include "FHSS.h"
+
+SX1280Driver Radio;
+
+uint8_t testdata[8] = {0x80};
+
+void ICACHE_RAM_ATTR TXdoneCallback()
+{
+    Serial.println("TXdoneCallback1");
+    //delay(1000);
```

Comment:
Need this?

Reply:
lets move the whole file to tests or something